fuel-tx/src/transaction/consensus_parameters.rs

```diff
@@ -24,6 +24,14 @@ const MAX_SIZE: u64 = 110 * 1024;
 
 #[derive(Debug)]
 pub struct SettingBlockTransactionSizeLimitNotSupported;
+#[cfg(feature = "std")]
+impl std::fmt::Display for SettingBlockTransactionSizeLimitNotSupported {
```

Comment:
1. use `core` instead of `std`
2. check if we can derive `Display` here

Reply:
Updated in https://github.com/FuelLabs/fuel-vm/pull/833/commits/1a1182436b235a27dce235171cadafe99b4e6dc8